src/components/PreviewCompatibleImage.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import Img from 'gatsby-image'
 
 const PreviewCompatibleImage = ({ imageInfo }) => {
-  const imageStyle = { borderRadius: '5px' }
+  const imageStyle = { borderRadius: '5px' , maxHeight:'800px', maxWidth:'800px'}
```

Comment:
Why is this here? It's nit used on the profile page, so can we just revert this line? I prefer to use the image processing graphql queries to set image sizes, as it ensures we aren't accidentally using a 3MB image file for a thumbnail...